Buildings/Experimental/RadiantControl/SlabTempSignal/OutputOnly/SlabSetCore.mo

```diff
@@ -0,0 +1,60 @@
+within Buildings.Experimental.RadiantControl.SlabTempSignal.OutputOnly;
+block SlabSetCore
+  "Determines slab temperature setpoint for core zones"
+  parameter Real TSlaCor(min=0,
+    final unit="K",
+    final displayUnit="K",
+    final quantity="Temperature")=294.3;
+
+  Controls.OBC.CDL.Interfaces.RealOutput TSlaSetCor
+    "Slab temperature setpoint for a core zone"
+    annotation (Placement(transformation(extent={{100,-10},{140,30}})));
+  Controls.OBC.CDL.Continuous.Sources.Constant TSlabCore(k=TSlaCor)

```

Comment:
Please try to stick to the 3-letter rule for instance names, similarly to how it is for any variable names. It is simply a convention in the library.